### PR TITLE
Added DrawChar method to Bitmap

### PR DESCRIPTION
**Pull request by mikeoliphant**
## Description
This change adds a DrawChar method to the managed Bitmap class.

## Motivation and Context
Previously, the only way to draw a single character was to draw a single-character string. This has a fair bit of extra overhead. The new DrawChar() method does this more efficiently.

## How Has This Been Tested?
Tested locally on M5Stack Core2.

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dev Containers (changes related with Dev Containers, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist

- [x] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [x] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).

### CODE REVIEW

**Comment by mikeoliphant** [comment]
Corresponding nanoFramework.Graphics PR is here: https://github.com/nanoframework/nanoFramework.Graphics/pull/107

File: src/nanoFramework.Graphics/Graphics/Core/Graphics.h [hunk 1]
```diff
@@ -245,8 +245,12 @@ struct DivHelper
         //       initY is the initial value of Y
         a = _a;
         b = _b;
+
```
Comment (Ellerbach) [inline src/nanoFramework.Graphics/Graphics/Core/Graphics.h:248]: please use explicit brackets, we're moving to have them in native as well.

File: src/nanoFramework.Graphics/Graphics/Native/nanoFramework_Graphics_nanoFramework_UI_Bitmap.cpp [hunk 2]
```diff
@@ -263,6 +263,32 @@ HRESULT Library_nanoFramework_Graphics_nanoFramework_UI_Bitmap::
...
+
+    CLR_GFX_Font *font;
+    CLR_RT_HeapBlock *pArgs;
+
+    pArgs = &(stack.Arg1());
+
+    NANOCLR_CHECK_HRESULT(Library_nanoFramework_Graphics_nanoFramework_UI_Font::GetFont(&pArgs[4], font));
+
+    bitmap->DrawChar(
+        pArgs[0].NumericByRef().u2,
```
Comment (Ellerbach) [inline src/nanoFramework.Graphics/Graphics/Native/nanoFramework_Graphics_nanoFramework_UI_Bitmap.cpp:283]: are you sure you have 4 arguments? From the definition, seems only 3

File: src/nanoFramework.Graphics/Graphics/Core/Graphics.h (hunk 1 above)
Comment (mikeoliphant) [reply src/nanoFramework.Graphics/Graphics/Core/Graphics.h:248]: Fixing now.

File: src/nanoFramework.Graphics/Graphics/Native/nanoFramework_Graphics.cpp [hunk 3]
```diff
@@ -977,7 +978,7 @@ static const CLR_RT_MethodHandler method_lookup[] =
 const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_nanoFramework_Graphics =
 {
     "nanoFramework.Graphics",
-    0xFA76FD3F,
+    0xD528A218,
     method_lookup,
     { 100, 0, 0, 4 }
```
Comment (josesimoes) [inline src/nanoFramework.Graphics/Graphics/Native/nanoFramework_Graphics.cpp:983]: Change in API calls for bumping the native assembly version too. Please do this here and on the C# lib too.

**Review by josesimoes — CHANGES_REQUESTED** [review]
Just the version needs to be bumped. All ood for the rest. 👍

File: src/nanoFramework.Graphics/Graphics/Native/nanoFramework_Graphics.cpp (hunk 3 above)
Comment (mikeoliphant) [reply src/nanoFramework.Graphics/Graphics/Native/nanoFramework_Graphics.cpp]: Done.

**Review by josesimoes — APPROVED** [review]
LGTM!

**Pull request merged**